docs/crictl.md

```diff
@@ -0,0 +1,172 @@
+CRICTL User Guide
+=================
+This document presumes you already have `containerd` and `cri-containerd`
+installed and running.
+
+This document is for developers who wish to debug, inspect, and manage their pods,
+containers, and container images.
+
+Before generating issues against this document, `containerd`, `cri-containerd`,
+or `crictl` please make sure the issue has not already been submitted.
+
+## Install crictl
+If you have not already installed crictl please install the version compatible
+with the `cri-containerd` you are using. If you are a user, your deployement
+should have installed crictl for you. If not, get it from your release tarball.
+The current version of crictl is specified [here](./hack/version). A helper
+command has been included to install the dependencies at the right version for
+developers:
+```shell
+$ make install.deps
+```
+* Note: The file named `/etc/crictl.yaml` is used to configure crictl
+so you don't have to repeatedly specify the runtime sock used to connect crictl
+to the container runtime:
+```
+# cat /etc/crictl.yaml
+runtime-endpoint: /var/run/dockershim.sock
+image-endpoint: /var/run/dockershim.sock
+timeout: 10
+debug: true
+```
+
+## Download and Inspect a Container Image
+```shell
+$ crictl pull busybox
+  ...
+$ crictl inspecti busybox
+```
+## Run a pod sandbox (using a config file)
+```shell
+$ cat sandbox-config.json
+{
+    "metadata": {
+        "name": "nginx-sandbox",
+        "namespace": "default",
+        "attempt": 1,
+        "uid": "hdishd83djaidwnduwk28bcsb"
+    },
+    "linux": {
+    }
+}
+
+$ crictl runs sandbox-config.json
+e1c83b0b8d481d4af8ba98d5f7812577fc175a37b10dc824335951f52addbb4e
+$ crictl sandboxes
+SANDBOX ID          CREATED             STATE               NAME                                                              NAMESPACE                                                ATTEMPT
```

Comment:
Check the preview, too many spaces between NAMESPACE and ATTEMPT

Reply:
done